frontend/src/features/admin-form/create/logic/components/LogicContent/InactiveLogicBlock/InactiveLogicBlock.tsx

```diff
@@ -98,6 +98,7 @@ export const InactiveLogicBlock = ({
         borderColor="neutral.300"
         cursor={isPreventEdit ? 'not-allowed' : 'pointer'}
```

Comment:
Can check if changing 'pointer' to 'auto' works as well? This might be able to allow us to have one single control on cursors.

Reply:
Might be able to change the one in the inactive step block as well - i am testing this on local